Makefile

```diff
@@ -4,6 +4,30 @@ export PATH := $(shell pwd)/tmp:$(PATH)
 .ONESHELL .PHONY: up update-box destroy-box remove-tmp clean example
 .DEFAULT_GOAL := up
 
+###################################
+########## Color Scheme ###########
+###################################
+ifneq (,$(findstring xterm,${TERM}))
+	BLACK        := $(shell tput -Txterm setaf 0)
+	RED          := $(shell tput -Txterm setaf 1)
+	GREEN        := $(shell tput -Txterm setaf 2)
+	YELLOW       := $(shell tput -Txterm setaf 3)
+	LIGHTPURPLE  := $(shell tput -Txterm setaf 4)
+	PURPLE       := $(shell tput -Txterm setaf 5)
+	BLUE         := $(shell tput -Txterm setaf 6)
+	WHITE        := $(shell tput -Txterm setaf 7)
+	RESET := $(shell tput -Txterm sgr0)
+else
+	BLACK        := ""
+	RED          := ""
+	GREEN        := ""
+	YELLOW       := ""
+	LIGHTPURPLE  := ""
+	PURPLE       := ""
+	BLUE         := ""
+	WHITE        := ""
+	RESET        := ""
+endif
```

Comment:
I assume this is because only xterm supports these colour commands? Couldn't we just rather use the colour definitions that was originally, but put them in variables like you did? Or did you encounter an error?

Love the variables btw, way way cleaner :D

Reply:
@pdmthorsrud :  Hmm, I did try to use the old definitions, but I was not able to make that work. I suppose with the correct syntax that would work yes.

Reply:
Also I think the main problem had to do with the use of [.ONESHELL](https://www.gnu.org/software/make/manual/html_node/One-Shell.html). This is why I added "#" at the start of the template_init command.

Reply:
Oh, I thought I replied on this one, sorry. 

I find it weird that oneshell was the problem, and adding a comment helped, because the `@` is wanted to surpress outputting the command itself, and is the correct syntax.

![image](https://user-images.githubusercontent.com/51820995/108479212-4b376500-724a-11eb-909f-ae52df6277ed.png)
https://opensource.com/article/18/8/what-how-makefile
![image](https://user-images.githubusercontent.com/51820995/108479284-630ee900-724a-11eb-84a2-562505379657.png)

Reply:
I see you changed the color-definitions. Resolving this one 👍